### DISCUSSION TITLE
Add note on each prompt version

### DISCUSSION THREAD

**Discussion by cubxxw** [langfuse/langfuse#2721] [Ideas]
### Describe the feature or potential improvement

Prompts in this interface, I want to prompt a few notes for the different prompt, is that OK? It could be a message or a note ？

### Additional information

This prompt I would like to have a note module or a comment module next to it that would do some prompt explanation 

![image](https://github.com/user-attachments/assets/5380638e-2aca-46eb-9fbd-2b07289d0b9b)

**Comment by marcklingen** [comment]
great idea, thank you!

**Comment by marcklingen** [comment]
@marliessophie added this, we forgot to update this thread: https://langfuse.com/changelog/2025-01-28-prompt-commit-messages

**Comment by cubxxw** [comment]
nice
